front/src/applications/editor/tools/routeEdition/components/WayPointInput.tsx

```diff
@@ -1,104 +1,96 @@
-import React, { useCallback, useContext, useEffect, useState } from 'react';
+import React, { FC, useCallback, useContext, useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import { useDispatch } from 'react-redux';
-import { GoTrash } from 'react-icons/go';
+import { GoPencil } from 'react-icons/go';
 import { FaMapMarkedAlt, FaTimesCircle } from 'react-icons/fa';
-import type { Position } from 'geojson';
-
-import type { EndPoint, WayPoint, WayPointEntity } from 'types';
+import { isNil } from 'lodash';
 
 import EditorContext from 'applications/editor/context';
+import { RouteEditionState, EndPointKeys } from 'applications/editor/tools/routeEdition/types';
```

Comment:
See https://osrd.fr/en/docs/guides/contribute/conventions/front/#import--export

```suggestion
import { type RouteEditionState, EndPointKeys } from 'applications/editor/tools/routeEdition/types';
```